KubeArmor/enforcer/appArmorProfile.go

```diff
@@ -25,12 +25,13 @@ func (ae *AppArmorEnforcer) ResolvedProcessWhiteListConflicts(processWhiteList *
 
 	for index, line := range *processWhiteList {
 		for source := range fromSources {
-			if strings.Contains(line, source) {
+			if strings.Contains(line, source) && line[:len(source)] == source {
```

Comment:
I dont think this check is correct.

If I have line=`/app/cartservice ix` and source=`/app` then the check will pass!

Reply:
Understood Rahul. Let me check again.